app/views/catalog/_numismatics_home_panel.html.erb

```diff
@@ -0,0 +1,12 @@
+<% if can?(:create, NumismaticIssue) %>
```

Comment:
@eliotjordan

Reply:
Oh I see. My local copy is not up to date. Let me pull in the latest changes.